app/views/services/index.html.erb

```diff
@@ -1,11 +1,3 @@
-
 <%= render 'shared/banner' %>
 
-<% @services.each do |service| %>
-<%= render 'shared/card-service' %>
-
-  <%= service.name %><br>
-  <%= service.content %><br>
-  <%= service.location %><br>
-  <%= service.price %><br>
-<% end %>
+<%= render 'shared/card-service' %>
```

Comment:
The variable service needs to be sent to the partial like:
<%= render 'shared/service_form**', service: @service** %>